src/products/mozilla_org/testing.md

```diff
@@ -52,28 +52,62 @@ You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.or
 * Test the page layout on the leading locally developed browser if available.
 * Test the page layout on mobile devices of major platforms.
 
-## When can I see the localized page on the production server?
+## Page activation
 
-Updated translations are pushed to the production server almost daily.
+Updated translations are pushed to the production server regularly throughout the day.
 
-When a brand new page is available for localization, it won’t be enabled in production until it’s fully localized. When existing pages receive updates with new strings, this new content won’t be displayed on production until localized, to avoid displaying a mix of English and localized text.
-
-In some cases, pages receive major updates that require a complete rewrite of the template. If this happens, the old template is kept online only for a short period of time and, when removed, it will cause the URL to redirect users to the English version.
+* When a brand new page is available for localization, it won’t be enabled on production until it’s at least 80% complete. This means, after it reaches this threshold, the page will be on production in English and target language.
```

Comment:
I don't understand the second part. The page is obviously available in English from the beginning, why mention it?

Reply:
When the page is brand new, there is an activation process and the locale page won't show until the page has 80% localized content. For updated pages, yes, the strings would show up right away, unless it is tagged - a scenario that is likely. Bedrock team has yet to address this.

Reply:
So, what you mean is that "English and target language" are displayed together? I didn't understand it at all. If that's correct, maybe "the page will be displayed on production potentially with English content mixed with translations, if completion level is not 100%".

Reply:
It will, not just potentially. The threadshold is 80%. Once you pass that point, if you don't finish it in one swoop, it will display a bilingual page for extended period of time until you get to it the next day or hours later.